modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -2782,24 +2782,49 @@ Usage:
 
 [source,console]
 ----
-maintenance:install [--database=["..."]] [--database-name=["..."]] \
-                    [--database-host=["..."]] [--database-user=["..."]] \
-                    [--database-pass=["..."]] [--database-table-prefix=["..."]] \
-                    [--admin-user=["..."]] [--admin-pass=["..."]] [--data-dir=["..."]]
+maintenance:install [--database=["..."]] [--database-connection-string=["..."]] \
+                    [--database-name=["..."]] [--database-host=["..."]] \
+                    [--database-user=["..."]] [--database-pass=["..."]] \ 
+                    [--database-table-prefix=["..."]] [--admin-user=["..."]] \
+                    [--admin-pass=["..."]] [--data-dir=["..."]]
 ----
 
 ===== Options
 
 [width="100%",cols="22%,70%",]
 |===
-| `--database`               | Supported database type (default: `sqlite`).
-| `--database-name`          | Name of the database.
-| `--database-host`          | Hostname of the database (default: `localhost`).
-| `--database-user`          | User name to connect to the database.
-| `--database-pass`          | Password of the database user.
-| `--database-table-prefix`  | Prefix for all tables (default: `oc_` ).
-| `--admin-user`             | Password of the admin account.
-| `--data-dir`               | Path to data directory (default: `/var/www/owncloud/data`).
+| `--database`                   
+| Supported database type (default: `sqlite`).
+| `--database-connection-string` 
+a| And Oracle-specific connection string. 
```

Comment:
I think this: `a| And Oracle` is mistyped and should be `|  A Oracle` 
If you look at how it is rendered, it makes it clear what is meant

Reply:
No, it's fine how it is. The `a` prefix indicates that [the cell contains AsciiDoc content](https://asciidoctor.org/docs/user-manual/#cell).

Reply:
> I think this: `a| And Oracle` is mistyped and should be `| A Oracle`
> If you look at how it is rendered, it makes it clear what is meant

That said, I see a grammatical mistake. Thanks for showing it to me.

Reply:
Fixed.